pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -314,3 +314,19 @@ def test_malformed_skipfooter(python_parser_only):
     msg = "Expected 3 fields in line 4, saw 5"
     with pytest.raises(ParserError, match=msg):
         parser.read_csv(StringIO(data), header=1, comment="#", skipfooter=1)
+
+
+def test_delimiter_with_usecols_and_parse_dates(python_parser_only):
```

Comment:
doesn't this not work in c-parser? why?

Reply:
Works for c too, not quite sure why I tested only for python. Moved it